ApplicationPattern.yaml

```diff
@@ -8605,6 +8635,160 @@ paths:
+    parameters:
+      - in: path
+        name: uuid
+        required: true
+        schema:
+          type: string
+          pattern: '^([a-z]{2,6})-([0-9]{1,2})-([0-9]{1,2})-([0-9]{1,2})-tcp-s-bm-000'
+          example: 'ro-2-0-0-tcp-s-bm-000'
+    get:
+      operationId: getTcpServerDescription
+      summary: 'Returns Description of TcpServer'
+      tags:
+        - TcpServer
+      security:
+        - basicAuth: []
+      responses:
+        '200':
+          description: 'Description provided'
+          content:
+            application/json:
+              schema:
+                type: object
+                required:
+                  - tcp-server-interface-1-0:description
+                properties:
+                  tcp-server-interface-1-0:description:
+                    type: string
+        '400':
+          $ref: '#/components/responses/responseForErroredOamRequests'
+        '401':
+          $ref: '#/components/responses/responseForErroredOamRequests'
+        '403':
+          $ref: '#/components/responses/responseForErroredOamRequests'
+        '404':
+          $ref: '#/components/responses/responseForErroredOamRequests'
+        '500':
+          $ref: '#/components/responses/responseForErroredOamRequests'
+        default:
+          $ref: '#/components/responses/responseForErroredOamRequests'
+    put:
+      operationId: putTcpServerDescription
+      summary: 'Documents Description of TcpServer'
+      tags:
+        - TcpServer
+      security:
+        - basicAuth: []
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              required:
+                - tcp-server-interface-1-0:description
+              properties:
+                tcp-server-interface-1-0:description:
+                  type: string
+      responses:
+        '204':
+          description: 'Local port documented'
+        '400':
+          $ref: '#/components/responses/responseForErroredOamRequests'
+        '401':
+          $ref: '#/components/responses/responseForErroredOamRequests'
+        '403':
+          $ref: '#/components/responses/responseForErroredOamRequests'
+        '404':
+          $ref: '#/components/responses/responseForErroredOamRequests'
+        '500':
+          $ref: '#/components/responses/responseForErroredOamRequests'
+        default:
+          $ref: '#/components/responses/responseForErroredOamRequests'  
+  /core-model-1-4:control-construct/logical-termination-point={uuid}/layer-protocol=0/tcp-server-interface-1-0:tcp-server-interface-pac/tcp-server-interface-configuration/protocol:
+    parameters:
+      - in: path
+        name: uuid
+        required: true
+        schema:
+          type: string
+          pattern: '^([a-z]{2,6})-([0-9]{1,2})-([0-9]{1,2})-([0-9]{1,2})-tcp-s-bm-000'
```

Comment:
Same proposal for the pattern as similar to the one provided for TcpServerDescription.
**Proposal :** 
Requesting to change the Pattern from , 
`pattern: '^([a-z]{2,6})-([0-9]{1,2})-([0-9]{1,2})-([0-9]{1,2})-tcp-s-bm-000'`
to 
`pattern: '^([a-z]{2,6})-([0-9]{1,2})-([0-9]{1,2})-([0-9]{1,2})-tcp-s-bm-00[0-1]`

Reply:
Also kindly include the same pattern to all the OAM services related to the tcp-server path parameters. Thanks.

Reply:
Changed as proposed.